infrastructure/networked_rds/variables.tf

```diff
@@ -25,6 +25,7 @@ variable stage {
 }
 variable region {
   description = "the aws region code where this is deployed; ex: 'us-west-2', 'us-east-1', 'us-east-2'"
+  default     = "us-east-2"
```

Comment:
for this project a better default will be "us-west-2"

Reply:
okay, I'll make that change.

Reply:
ah it's ok you can disregard